antlr4/PostgreSQLParser.g4

```diff
@@ -1393,6 +1401,7 @@ expr
     | expr op=IS (bool_expr | NULL)
     | expr op=(ISNULL | NOTNULL)
     | expr IS NOT? DISTINCT FROM expr
+    | expr IS NOT? NULL
```

Comment:
I don't have great insight into the slowdown in the select test cases, but I don't see anything effecting those tests in this PR except for `expr`. This change to `expr` seems simple and safe, but can we compare the timing of the tests with this change and with it commented? I know it sounds silly to test, but I do wonder...

As a side note, after we've reached basic correctness in the tests, `expr` is one of those rules that needs a top to bottom review. It has things like `expr op=IS (bool_expr | NULL)` along with `expr IS NOT? NULL`, and they be consolidated along with other things.

Reply:
I thought about this some more and it brings back an old memory when I initially implemented this. I suspect this rule is triggering excess recursion through the case where `NOT` is include and the case where `NOT` is not included, which may trigger more recursion down the tree.

Reply:
I has simplify `EXISTS OPEN_PAREN select_stmt CLOSE_PAREN`. Now tests run for ~ 1 minute. Maybe I will reduce the time of testing in the near future.